app/views/reviews/new.html.erb

```diff
@@ -1,8 +1,5 @@
-<h2>Add New Review</h2>
+<h1>Add New Review</h1>
```

Comment:
<h1>Add New Review</h1>
we should keep our header tag size uniform for each CRUD page